packages/eslint-config-edx/.eslintrc.json

```diff
@@ -6,14 +6,19 @@
         "jasmine": true,
         "jquery": true
     },
-    "extends": "airbnb-base",
+    "extends": "airbnb",
     "parserOptions": {
-        "ecmaVersion": 8
+        "ecmaVersion": 8,
+        "ecmaFeatures": {
+          "jsx": true
+        }
     },
     "plugins": [
         "dollar-sign"
     ],
     "rules": {
-        "dollar-sign/dollar-sign": ["error", "ignoreProperties"]
+        "dollar-sign/dollar-sign": ["error", "ignoreProperties"],
+        "react/no-array-index-key": "off",
+        "strict": "off"
```

Comment:
I'm surprised that AirBnB don't have this off since it seems that it is irrelevant in ES2015:

http://eslint.org/docs/rules/strict

I don't understand why they say that they rely on Babel, since don't they lint files pre-Babel?

https://github.com/airbnb/javascript/blob/a2a35a8dbadc229c402b9752412d22ce5c42f33b/packages/eslint-config-airbnb-base/rules/strict.js#L4

Anyway, this rule makes sense to me.